backend/node/base.py

```diff
@@ -240,6 +274,13 @@ async def _run_genvm(
         snapshot_view = _SnapshotView(
             self.contract_snapshot, self.contract_snapshot_factory, readonly
         )
+
+        # FIXME(core-team): please provide valid date
+        transaction_datetime = datetime.datetime.fromisoformat(
```

Comment:
here if you pass correct `datetime.datetime` everything will work. I am not a high skilled full stack developer, so I left database patching to you, sorry @cristiam86

Reply:
@kp2pml30 sorry but I'm not sure what do you mean with database patching. Here we should pass the current timestamp on ISO format in UTC timezone.
Could it be: `transaction_datetime  = datetime.now(timezone.utc)` ?